kgforge/core/archetypes/store.py

```diff
@@ -369,8 +372,9 @@ def sparql(
             if self.model_context is not None and rewrite
             else query
         )
-        qr = _replace_in_sparql(qr, "LIMIT", limit, 100, r" LIMIT \d+")
-        qr = _replace_in_sparql(qr, "OFFSET", offset, 0, r" OFFSET \d+")
+        if rewrite:
```

Comment:
This is linked to https://github.com/BlueBrain/nexus-forge/issues/262 , to ensure that the parameters are only replaced when `rewrite` is `True`.